packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -314,7 +314,16 @@ export class StripeWebhookHandler extends StripeHandler {
       );
       return;
     }
-    await this.paypalHelper.issueRefund(invoice, transactionId);
+    try {
+      await this.paypalHelper.issueRefund(invoice, transactionId);
+    } catch (error) {
+      if (error instanceof RefusedError) {
+        await this.stripeHelper.updateInvoiceWithPaypalRefundReason(
+          invoice,
+          error.longMessage
+        );
+      }
```

Comment:
If we don't re-throw here for all the other types of errors, then this behavior is quite different than what we had previously.

Reply:
Ah, yes, added that.